vsts-ci.yml

```diff
@@ -20,6 +20,12 @@ queue:
   timeoutInMinutes: 120
 
 steps:
+- powershell: |
```

Comment:
Shouldn't the `CloudBuild: buildNumber: enabled: true` section do this already?

Reply:
Pertty sure the nbgv targets already do this, right?

Reply:
Not sure... vswhere had this and buildnumber enabled as false. So I could just be deleting this step and call it good